packages/core/src/menu/MenuTrigger.tsx

```diff
@@ -11,39 +17,42 @@ export interface MenuTriggerProps {
   children?: ReactNode;
 }
 
-export function MenuTrigger(props: MenuTriggerProps) {
-  const { children } = props;
+export const MenuTrigger = forwardRef<HTMLElement, MenuTriggerProps>(
+  function MenuTrigger(props, ref) {
+    const { children } = props;
 
-  const { getReferenceProps, refs, setFocusInside, focusInside, openState } =
-    useMenuContext();
-  const { setFocusInside: setFocusInsideParent } = useMenuPanelContext();
+    const { getReferenceProps, refs, setFocusInside, focusInside, openState } =
+      useMenuContext();
+    const { setFocusInside: setFocusInsideParent } = useMenuPanelContext();
 
-  const handleRef = useForkRef(
-    getRefFromChildren(children),
-    refs?.setReference,
-  );
+    const handleFloatingRef = useForkRef(
+      getRefFromChildren(children),
+      refs?.setReference,
+    );
+    const handleRef = useForkRef(handleFloatingRef, ref);
 
-  if (!children || !isValidElement<{ ref?: Ref<unknown> }>(children)) {
-    // Should we log or throw error?
-    return <>{children}</>;
-  }
+    if (!children || !isValidElement<{ ref?: Ref<unknown> }>(children)) {
```

Comment:
NIT 
 * The trigger element for the menu. This should be a single child element and accept a ref.
 
 We cover in the props that the child element should receive a ref... Wonder if that is enough and needs to be in usage  docs ?